modules/core/remove_orphaned.py

```diff
@@ -102,13 +96,14 @@ def move_orphan(self, file):
         util.move_files(src, dest, True)
         return os.path.dirname(file).replace(self.root_dir, self.remote_dir)
 
-
-def get_full_path_of_torrent_files(torrent_files_and_save_path):
-    torrent_files, save_path = torrent_files_and_save_path
-    fullpath_torrent_files = []
-    for file in torrent_files:
-        fullpath = os.path.join(save_path, file)
-        # Replace fullpath with \\ if qbm is running in docker (linux) but qbt is on windows
-        fullpath = fullpath.replace(r"/", "\\") if ":\\" in fullpath else fullpath
-        fullpath_torrent_files.append(fullpath)
-    return fullpath_torrent_files
+    def get_full_path_of_torrent_files(self, torrent):
+        torrent_files = map(lambda dict: dict.name, torrent.files)
```

Comment:
Would this work with torrents with the same name? i.e. cross-seeds? I guess the question really is if it's safe to assume the save_path is the same for all torrents with the same name.

Reply:
All this is doing is getting the full paths of all files within a torrent in order to know which files in root_dir are orphans, I don't see why a torrent hash would matter here. The first comment was under the impression that this was making a dictionary that maps files to torrents which it isn't